graylog2-web-interface/src/views/logic/SearchError.js

```diff
@@ -21,3 +21,30 @@ export default class SearchError {
 
   get type() { return this._state.type; }
 }
+
+export class ResultWindowLimitError extends SearchError {
```

Comment:
This is the approach to move the error logic from MessageList component to the `SearchError` class. The goal is extending the description if the error is a result window limit error. I was not so sure about, where to create the different errors (currently in the SearchResult constructor) and how to deal with the prop result. Should it be a part of the class? Right now it is not a part of the class and I am only accessing it when constructing the class. That's why I am using static classes to extend the description.